inc/class-cachify.php

```diff
@@ -1334,6 +1334,18 @@ public static function set_cache( $data ) {
 
 		/* Save? */
 		if ( $should_cache ) {
+			/**
+			 * Filters the buffered data itself
+			 *
+			 * @since 2.3.2
```

Comment:
Definitely out of scope for a patch release, should be 2.4 or whatever point release this will be merged into.